### PR TITLE
[DO NOT MERGE] Testing AccessLint

### PR DESCRIPTION
**Pull request by steveswinsburg**
Expect comments from AccessLint

### CODE REVIEW

File: accesslint-test.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<img src="logo.png" />
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply accesslint-test.html]: Looks good @steveswinsburg !

[the remark this answers is not in the document]
Comment (SedueRey) [reply accesslint-test.html]: Awesome!

**Comment by jonespm** [comment]
Closing this, re-open if you still need to test.

**Comment by steveswinsburg** [comment]
It was more a demo than anything :)

**Pull request closed without merging**